doc/api/errors.md

```diff
@@ -795,6 +796,76 @@ Used when an attempt is made to launch a Node.js process with an unknown
 by errors in user code, although it is not impossible. Occurrences of this error
 are most likely an indication of a bug within Node.js itself.
 
+<a id="ERR_MULTIPLE_CALLBACK"></a>
+### ERR_MULTIPLE_CALLBACK
+
+Used when a callback is called more than once.
+
+<a id="ERR_ASSERTION"></a>
+### ERR_ASSERTION
+
+Used as special type of error that can be triggered whenever Node.js detects an
+exceptional logic violation that should never occur. These are raised typically
+by the `assert` module.
+
+<a id="ERR_CPU_USAGE"></a>
+### ERR_CPU_USAGE
+
+Used when the native call from process.cpuUsage can't be processed properly.
+
+<a id="ERR_HTTP_HEADERS_SENT"></a>
+### ERR_HTTP_HEADERS_SENT
+
+Used in case the headers were already sent and another attempt is made to add
+more headers.
+
+<a id="ERR_INVALID_CHAR"></a>
+### ERR_INVALID_CHAR
+
+Used in e.g. http if invalid characters are detected in headers.
+
+<a id="ERR_HTTP_INVALID_STATUS_CODE"></a>
+### ERR_HTTP_INVALID_STATUS_CODE
+
+Used for nonexisting status codes.
+
+<a id="ERR_INVALID_ARRAY_LENGTH"></a>
+### ERR_INVALID_ARRAY_LENGTH
+
+Used in case a array is not of the expected length or in a valid range.
+
+<a id="ERR_INVALID_FD"></a>
+### ERR_INVALID_FD
+
+Used in case a FD entry is not valid (i.e. negative).
+
+<a id="ERR_PARSE_HISTORY_DATA"></a>
+### ERR_PARSE_HISTORY_DATA
+
+Used in the `repl` if you use the old history file and an error occurred while
+trying to read and parse it.
+
+<a id="ERR_STREAM_HAS_STRINGDECODER"></a>
+### ERR_STREAM_HAS_STRINGDECODER
+
+TODO: Add description
+
+<a id="ERR_TRANSFORM_ALREADY_TRANSFORMING"></a>
+### ERR_TRANSFORM_ALREADY_TRANSFORMING
+
+Used when stream transform done is called even though it is still in process.
+
+<a id="ERR_HTTP_TRAILER_INVALID"></a>
+### ERR_HTTP_TRAILER_INVALID
+
+Used when the `Trailer` header is set even though the encoding does not support
+that.
```

Comment:
this is misleading. It's not the `encoding` that matters, it is the *transfer encoding* ... specifically,